sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Constants.cs

```diff
@@ -6,6 +6,6 @@ namespace Microsoft.Azure.WebJobs.Extensions.Storage.Common
     internal static class Constants
     {
         public const string DateTimeFormatString = "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffK";
-        public const int MaxDepthDefault = 128;
+        public const int MaxDepthDefault = 4;
```

Comment:
Is this large enough now? I only mentioned "4" in email - with a note saying implying it was entirely random - as an example. Make sure it's large enough for your needs if you know the shape of the response JSON.

Unblocking since I'm OOF starting tomorrow, but make sure this value works for you - whatever it may be.

Reply:
We can talk more offline about this value, I think we're still going back and forth about what the best value is. I put it back to 128 cause that's the default for the NewtonJson package.